src/main/cpp/src/get_json_object.cu

```diff
@@ -1179,7 +921,7 @@ __device__ thrust::pair<bool, size_t> get_json_object_single(
   json_generator generator((out_buf == nullptr || out_buf_size == 0) ? nullptr : out_buf);
 
   bool const success = evaluate_path(
-    j_parser, generator, write_style::raw_style, path_commands_ptr, path_commands_size);
+    j_parser, generator, write_style::RAW, {path_commands.data(), path_commands.size()});
 
   if (nullptr == out_buf && !success) {
```

Comment:
Remove `nullptr == out_buf`?
On first phase the out_buf is null, on second phase it's not null.

Reply:
done.